boutdata/collect.py

```diff
@@ -1215,7 +1223,8 @@ def create_cache(path, prefix):
 
     Returns
     -------
-    namedtuple : (list of str, bool, str, list of :py:obj:`~boututils.datafile.DataFile`)
+    namedtuple : (list of str, bool, str,
+                  list of :py:obj:`~boututils.datafile.DataFile`)
```

Comment:
Would this line break mess up the rendered documentation?

Reply:
Didn't think of that before, but I've checked now in another project that builds docs with sphinx, and it's fine.